cl/api/pagination.py

```diff
@@ -62,3 +74,194 @@ class MediumAdjustablePagination(ShallowOnlyPageNumberPagination):
 
 class BigPagination(ShallowOnlyPageNumberPagination):
     page_size = 300
+
+
+class ESCursorPagination(BasePagination):
+    """Custom pagination class to handle ES cursor pagination, based in the ES
+    search_after param.
+    """
+
+    def __init__(self):
+        self.page_size = settings.SEARCH_API_PAGE_SIZE
+        self.request = None
+        self.es_list_instance = None
+        self.results_count_exact = None
+        self.results_count_approximate = None
+        self.child_results_count = None
+        self.results_in_page = None
+        self.base_url = None
+        self.cursor = None
+        self.search_type = None
+        self.cursor_query_param = "cursor"
+        self.invalid_cursor_message = "Invalid cursor"
```

Comment:
Since the `__init__` method doesn't accept any arguments and only assigns values to variables, I believe we can simplify the code by moving these assignments directly to the class level as class variables.

Reply:
Yeah, you're right. I have moved them to class variables. The only one that needed to be within the `__init__` was `self.page_size` for testing purposes. However, it's better to use `settings.SEARCH_API_PAGE_SIZE` directly in the code and remove `__init__`.